FirebaseMessaging.podspec

```diff
@@ -57,9 +57,15 @@ device, and it is completely free.
 
   s.test_spec 'unit' do |unit_tests|
     unit_tests.scheme = { :code_coverage => true }
-    unit_tests.platforms = {:ios => '10.0', :osx => '10.12', :tvos => '10.0'}
-    unit_tests.source_files = 'FirebaseMessaging/Tests/UnitTests*/*.{m,h,swift}'
-    unit_tests.requires_app_host = true
+    unit_tests.platforms = {:ios => '10.0', :osx => '10.12', :tvos => '10.0', :watchos => '7.4'}
```

Comment:
I don't think that any other lines need to be changed than this one, since they replicate for all the platforms.

Reply:
You are right, I was playing around the podspec and forgot to cleanup.
I also updated the correct issue number.